src/Microsoft.NetCore.Analyzers/UnitTests/Tasks/DoNotCallBeginInvokeOnDelegateTests.cs

```diff
@@ -0,0 +1,663 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Testing;
+using Microsoft.CodeAnalysis.Testing.Verifiers;
+using Xunit;
+
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
```

Comment:
📝 Use `CSharpSecurityCodeFixVerifier` for analyzers that report diagnostics in generated code. It applies `TestBehaviors.SkipGeneratedCodeCheck` as part of the default test setup.

Reply:
Thank you. I didn't notice that class.